scripts/gameplay/Health.gd

```diff
@@ -0,0 +1,15 @@
+class_name Health
+extends Node
+
+signal death()
+
+@export var MaxHealth = 1.0
```

Comment:
max_health (try to follow https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html)